homeworks/01-bubble-sort/src/components/Items.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { TIMEOUT_TIME } from "../helpers/constants";
+import { GameStatus, SortStateStatus } from "../helpers/enums";
+import { ISortState } from "../interfaces/componentInterfaces";
+import { getSortState } from "../models/SortState";
+import { getInitialRandomSet, swapItems } from "../services/itemsService";
+import { getRandomSet } from "../services/randomService";
+import { ItemsProps, ItemsState } from "../types/componentTypes";
+import { Item } from "./Item";
+
+export class Items extends React.Component<ItemsProps, ItemsState> {
+    private _sortIntervalId: NodeJS.Timeout | null = null;
```

Comment:
`NodeJS.Timeout` is incorrect. Try `window.setTimeout` explicitely.

Reply:
It is just the type of the return value of the window.setTimeout function

Reply:
I see what you mean. I must use window.setTimeout instead of just setTimeout. As I understand it depends on execution env (node.js or browser)